### DISCUSSION TITLE
How to create darcy dataset on different resolution?

### DISCUSSION THREAD

**Discussion by Karn3003** [neuraloperator/neuraloperator#345] [Announcements]
Hi, I want to generate a darcy dataset on different resolutions. Can you please help me with the script for generating the dataset?

Thanks

**Comment by Hrrsmjd** [comment]
A quick option would be to download the Darcy-Flow dataset from [PDEBench ](https://github.com/pdebench/PDEBench) and just downsample to lower resolutions.

**Comment by MaxGhi8** [comment]
First of all the dataset provided has 421 points per direction, so if you want a coarser resolution you can simply subsample this one. Otherwise you can go to the branch `master` of this repository and then got to the folder `data_generation/darcy`, here you can find all the code for generating both inputs and outputs that you want.
